src/codec.rs

```diff
@@ -408,12 +420,32 @@ mod feature_wrapper_bytes {
 	impl EncodeLike<Bytes> for Vec<u8> {}
 }
 
+/// `Input` implementation optimized for decoding `bytes::Bytes`.
 #[cfg(feature = "bytes")]
-struct BytesCursor {
+pub struct BytesCursor {
```

Comment:
Why are you making it pub? Then we have to include it our public SemVer API.

Reply:
Reverted the change. Not applicable anymore.